pkg/logger/logger.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+import (
+	"fmt"
+	"go.uber.org/zap/zapcore"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"strings"
+)
+
+func SetLogLevel(logLevel string) (zap.Opts, error) {
+	var level zapcore.Level
+	switch strings.ToUpper(logLevel) {
+	case "TRACE":
```

Comment:
Are we sure we want to support trace? We have only talked about debug and better info

Reply:
As per the roman vote, trace has been expurgated from our code!